openreview/conference/helpers.py

```diff
@@ -275,7 +275,9 @@ def get_review_stage(request_forum):
         release_to_reviewers = release_to_reviewers,
         email_pcs = (request_forum.content.get('email_program_chairs_about_reviews', '').startswith('Yes')),
         additional_fields = review_form_additional_options,
-        remove_fields = review_form_remove_options
+        remove_fields = review_form_remove_options,
+        rating_field_name=request_forum.content.get('review_rating_field_name', 'rating').strip().lower().replace(" ", "_"),
```

Comment:
I'm not sure about replacing the spaces with _ here. The review_rating_field_name should always match with the field name described in Additional Review field options. 

If they define "Final Rating" as a key of the json then the value in review_rating_field_name should keep as "Final Rating" otherwise the UI won't find this value in the content.